README.md

```diff
@@ -268,58 +248,33 @@ The following launches 4 replicas of the the same ``hello_world`` model.
 # hello_ensemble.py
 from smartsim import Experiment
 
-exp = Experiment("hello_world_batch", launcher="slurm")
+exp = Experiment("hello_world_batch", launcher="auto")
 
 # define resources for all ensemble members
-sbatch = exp.create_batch_settings(nodes=4, time="00:10:00", account="12345-Cray")
-sbatch.set_partition("premium")
+batch = exp.create_batch_settings(nodes=4, time="00:10:00", account="12345-Cray")
+batch.set_queue("premium")
 
 # define how each member should run
-srun = exp.create_run_settings(exe="echo", exe_args="Hello World!")
-srun.set_nodes(1)
-srun.set_tasks(32)
-
-ensemble = exp.create_ensemble("hello_world", batch_settings=sbatch,
-                               run_settings=srun, replicas=4)
+run = exp.create_run_settings(exe="echo", exe_args="Hello World!")
+run.set_tasks(60)
+run.set_tasks_per_node(20)
+
+ensemble = exp.create_ensemble("hello_world",
+                               batch_settings=batch,
+                               run_settings=run,
+                               replicas=4)
 exp.start(ensemble, block=True, summary=True)
 
 print(exp.get_status(ensemble))
 ```
-```bash
-# on Slurm system
-python hello_ensemble.py
-```
-
-
-Here is the same example, but for PBS using ``AprunSettings`` for running with ``aprun``.
-``MpirunSettings`` could also be used in this example as openMPI supported by all the
-launchers within SmartSim.
-
-```python
-# hello_ensemble_pbs.py
-from smartsim import Experiment
-
-exp = Experiment("hello_world_batch", launcher="pbs")
-
-# define resources for all ensemble members
-qsub = exp.create_batch_settings(nodes=4, time="00:10:00",
-                                 account="12345-Cray", queue="cl40")
-
-# define how each member should run
-aprun = exp.create_run_settings(exe="echo", exe_args="Hello World!")
-aprun.set_tasks(32)
-
-ensemble = exp.create_ensemble("hello_world", batch_settings=qsub,
-                                run_settings=aprun, replicas=4)
-exp.start(ensemble, block=True, summary=True)
 
-print(exp.get_status(ensemble))
-```
 ```bash
-# on PBS system
-python hello_ensemble_pbs.py
+python hello_ensemble.py
 ```
 
+Similar to the interactive example, this same script will run on a SLURM, PBS, LSF,
+or Cobalt system as the ``launcher`` is set to `auto` in the Experiment
```

Comment:
I noticed some other formatting of Experiment as ``Experiment``.  Maybe check for consistency?